packages/leancode_contracts_generator/lib/src/statements/dto_handler.dart

```diff
@@ -32,6 +32,16 @@ class DtoHandler extends StatementHandler {
       if (typeDescriptor.genericParameters.isNotEmpty) {
         b.methods.clear();
       }
+
+      b.fields.add(
```

Comment:
How it will work with generic DTOs, e.g. `SortedQuery`/`PaginatedResult`?

Reply:
For generics, this is what get's generated: (so, just the name, without type params)
```dart
@ContractsSerializable(genericArgumentFactories: true)
class DTO2<T> with EquatableMixin {
  DTO2();

  factory DTO2.fromJson(
    Map<String, dynamic> json,
    T Function(Object?) fromJsonT,
  ) => _$DTO2FromJson(json, fromJsonT);

  List<Object?> get props => [];

  static const fullName = 'Notifications.Generic.DTO2';
}
```

Reply:
> Also, what if there is fullName field in the DTO?

There'll be a conflict. We can use `fullName$` for this static field